### PR TITLE
run-archive-node: add docs on how to run an archive node

### PR DESCRIPTION
**Pull request by ptrus**
TODO:
- [x] add a note that archive node is currently only supported in `21.x` 
- [x] update the config section a bit (e.g. other config settings should remain as they were. if node has runtime state, runtimes should be configured, etc...)

### CODE REVIEW

File: docs/general/run-a-node/set-up-your-node/run-archive-node.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+You can start the node by running the following command:
+
+```bash
+oasis-node --config /node/etc/config.yml
+```
+
+### Archive node status
+
+To ensure node is running in archive node, run the following command:
```
Comment (kostko) [inline docs/general/run-a-node/set-up-your-node/run-archive-node.md]:
```suggestion
To ensure the node is running in archive mode, run the following command:
```

File: docs/general/run-a-node/set-up-your-node/run-archive-node.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+```bash
+oasis-node control status --address /node/data/internal.sock
+```
+
+Output should report `archive` consensus mode status:
+
+```json
+{
+  // other fields emitted ...
```
Comment (matevz) [inline docs/general/run-a-node/set-up-your-node/run-archive-node.md]:
```suggestion
  // other fields omitted ...
```

File: docs/general/run-a-node/set-up-your-node/run-archive-node.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+```
+
+:::info
+
+Keep other settings unchanged from the full-node. For example in order to
+serve archived runtime state, the node needs to have the runtime configured and have
+the state present.
+
+:::
+
```
Comment (matevz) [inline docs/general/run-a-node/set-up-your-node/run-archive-node.md:49]:
```suggestion
```

File: docs/general/run-a-node/set-up-your-node/run-archive-node.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+# Run an Archive Node
+
+:::info
+
+Archive node support is currently only available in the [21.3.13](https://github.com/oasisprotocol/oasis-core/releases/tag/v21.3.13) release
+for use with the Cobalt (pre Damask upgrade) archived state.
+
+:::
+
+
```
Comment (matevz) [inline docs/general/run-a-node/set-up-your-node/run-archive-node.md]:
```suggestion
```

**Review by matevz — CHANGES_REQUESTED** [review]
Can you also limit the line length to 80 characters.

File: docs/general/run-a-node/set-up-your-node/run-archive-node.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+```bash
+oasis-node --config /node/etc/config.yml
+```
+
+### Archive node status
+
+To ensure the node is running in archive mode, run the following command:
+
+```bash
+oasis-node control status --address /node/data/internal.sock
```
Comment (matevz) [inline docs/general/run-a-node/set-up-your-node/run-archive-node.md]:
```suggestion
oasis-node control status -a unix:/node/data/internal.sock
```

**Review by matevz — APPROVED** [review]
Just a small nit, otherwise ok.

**Pull request merged**